redis-cluster/start.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+
+redis-server /usr/local/etc/redis/redis.conf --port 6391 --cluster-config-file nodes-6391.conf
```

Comment:
How does this work? From where does it fetch `nodes-6391.conf`?

Reply:
From Redis config:

> Every cluster node has a cluster configuration file. This file is not
> intended to be edited by hand. It is created and updated by Redis nodes.
> Every Redis Cluster node requires a different cluster configuration file.
> Make sure that instances running in the same system do not have
> overlapping cluster configuration file names.